CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Stylable Components
+
+## Branch naming
+We use scheme in branch naming.
+It should contain branch author user name at first and branch name divided by `/`, and optional number of the issue. All words in branch name are separated by underscore: `_`
+`username/own_theme_park_42`
```

Comment:
I like `username/issue_num_if_exists/branch_name`... much easier for me to find branches that way